packages/sortable-behavior-bundle/composer.json

```diff
@@ -18,6 +18,7 @@
     "homepage": "https://github.com/runroom/RunroomSortableBehaviorBundle",
     "require": {
         "php": "^7.4 || ^8.0",
+        "doctrine/annotations": "^1.13 || ^2.0",
```

Comment:
Not completely sure about this one, to me it should be require dev. Usage of the trait is not required and it also has attributes, so...